simplesat/dependency_solver.py

```diff
@@ -37,10 +61,21 @@ def repository_from_requirements(repositories, requirements, modifiers=None):
 
 
 def repository_is_consistent(repository, modifiers=None):
-    """ Return True if every package in the repository can be installed
-    simultaneously, otherwise False.
-
-    If `modifiers` are not None, use them when resolving requirements.
+    """
+    Return True if all packages in a repository can be installed together.
+
```

Comment:
We should mention this only really makes sense for repositories representing installed packages (e.g. no more than 1 version of any package in it).